src/LightGraphs.jl

```diff
@@ -86,9 +86,6 @@ CombinatorialAdjacency, non_backtracking_matrix, incidence_matrix,
 nonbacktrack_embedding, Nonbacktracking,
 contract,
 
-# astar
-a_star,
```

Comment:
(This was one of I think three non-whitespace changes I made [prior to our IM conversation last night]).